Class "Auth" not found
#0 C:\xampp\htdocs\intermodular\app\Core\Response.php(15): require()
#1 C:\xampp\htdocs\intermodular\app\Core\helpers.php(86): App\Core\Response::view('home.index', Array)
#2 C:\xampp\htdocs\intermodular\public\index.php(17): view('home.index')
#3 {main}Class "Auth" not found
#0 C:\xampp\htdocs\intermodular\app\Core\Response.php(15): require()
#1 C:\xampp\htdocs\intermodular\app\Core\helpers.php(86): App\Core\Response::view('home.index', Array)
#2 C:\xampp\htdocs\intermodular\public\index.php(17): view('home.index')
#3 {main}Class "Auth" not found
#0 C:\xampp\htdocs\intermodular\app\Core\Response.php(15): require()
#1 C:\xampp\htdocs\intermodular\app\Core\helpers.php(86): App\Core\Response::view('home.index', Array)
#2 C:\xampp\htdocs\intermodular\public\index.php(17): view('home.index')
#3 {main}[2025-05-16 16:42:55] ERROR: Class "Auth" not found
File: C:\xampp\htdocs\intermodular\resources\views\layouts\app.php Line: 12
Stack trace:
#0 C:\xampp\htdocs\intermodular\app\Core\Response.php(47): require()
#1 C:\xampp\htdocs\intermodular\app\Core\helpers.php(92): App\Core\Response::view('home.index', Array)
#2 C:\xampp\htdocs\intermodular\public\index.php(41): view('home.index')
#3 {main}

[2025-05-16 16:44:03] ERROR: Class "Auth" not found
File: C:\xampp\htdocs\intermodular\resources\views\layouts\app.php Line: 12
Stack trace:
#0 C:\xampp\htdocs\intermodular\app\Core\Response.php(47): require()
#1 C:\xampp\htdocs\intermodular\app\Core\helpers.php(92): App\Core\Response::view('home.index', Array)
#2 C:\xampp\htdocs\intermodular\public\index.php(41): view('home.index')
#3 {main}

[2025-05-16 16:44:22] ERROR: Class "Auth" not found
File: C:\xampp\htdocs\intermodular\resources\views\layouts\app.php Line: 12
Stack trace:
#0 C:\xampp\htdocs\intermodular\app\Core\Response.php(47): require()
#1 C:\xampp\htdocs\intermodular\app\Core\helpers.php(92): App\Core\Response::view('home.index', Array)
#2 C:\xampp\htdocs\intermodular\public\index.php(41): view('home.index')
#3 {main}

[2025-05-16 16:44:25] ERROR: Class "Auth" not found
File: C:\xampp\htdocs\intermodular\resources\views\layouts\app.php Line: 12
Stack trace:
#0 C:\xampp\htdocs\intermodular\app\Core\Response.php(47): require()
#1 C:\xampp\htdocs\intermodular\app\Core\helpers.php(92): App\Core\Response::view('home.index', Array)
#2 C:\xampp\htdocs\intermodular\public\index.php(41): view('home.index')
#3 {main}

SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo for sql111.infinityfree.com failed: Host desconocido. 
#0 C:\xampp\htdocs\intermodular\app\Core\DB.php(19): PDO->__construct('mysql:host=sql1...', 'if0_38694634', Object(SensitiveParameterValue))
#1 C:\xampp\htdocs\intermodular\app\Core\DB.php(97): App\Core\DB::connection()
#2 C:\xampp\htdocs\intermodular\app\Core\DB.php(33): App\Core\DB::prepare('SELECT * FROM c...', Array)
#3 C:\xampp\htdocs\intermodular\app\Core\DB.php(66): App\Core\DB::select('App\\Models\\Clie...', 'SELECT * FROM c...', Array)
#4 C:\xampp\htdocs\intermodular\app\Core\QueryBuilder.php(114): App\Core\DB::selectOne('App\\Models\\Clie...', 'SELECT * FROM c...', Array)
#5 C:\xampp\htdocs\intermodular\app\Core\Auth.php(16): App\Core\QueryBuilder->first()
#6 C:\xampp\htdocs\intermodular\app\Http\Controllers\AuthController.php(24): App\Core\Auth::attempt(Array)
#7 C:\xampp\htdocs\intermodular\public\auth\login.php(30): App\Http\Controllers\AuthController->login(Object(App\Core\Request))
#8 {main}SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\intermodular\app\Core\DB.php(19): PDO->__construct('mysql:host=loca...', 'root', Object(SensitiveParameterValue))
#1 C:\xampp\htdocs\intermodular\app\Core\DB.php(97): App\Core\DB::connection()
#2 C:\xampp\htdocs\intermodular\app\Core\DB.php(33): App\Core\DB::prepare('SELECT * FROM c...', Array)
#3 C:\xampp\htdocs\intermodular\app\Core\DB.php(66): App\Core\DB::select('App\\Models\\Clie...', 'SELECT * FROM c...', Array)
#4 C:\xampp\htdocs\intermodular\app\Core\QueryBuilder.php(114): App\Core\DB::selectOne('App\\Models\\Clie...', 'SELECT * FROM c...', Array)
#5 C:\xampp\htdocs\intermodular\app\Core\Auth.php(16): App\Core\QueryBuilder->first()
#6 C:\xampp\htdocs\intermodular\app\Http\Controllers\AuthController.php(24): App\Core\Auth::attempt(Array)
#7 C:\xampp\htdocs\intermodular\public\auth\login.php(30): App\Http\Controllers\AuthController->login(Object(App\Core\Request))
#8 {main}SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\intermodular\app\Core\DB.php(19): PDO->__construct('mysql:host=loca...', 'root', Object(SensitiveParameterValue))
#1 C:\xampp\htdocs\intermodular\app\Core\DB.php(97): App\Core\DB::connection()
#2 C:\xampp\htdocs\intermodular\app\Core\DB.php(33): App\Core\DB::prepare('SELECT * FROM c...', Array)
#3 C:\xampp\htdocs\intermodular\app\Core\DB.php(66): App\Core\DB::select('App\\Models\\Clie...', 'SELECT * FROM c...', Array)
#4 C:\xampp\htdocs\intermodular\app\Core\QueryBuilder.php(114): App\Core\DB::selectOne('App\\Models\\Clie...', 'SELECT * FROM c...', Array)
#5 C:\xampp\htdocs\intermodular\app\Core\Auth.php(16): App\Core\QueryBuilder->first()
#6 C:\xampp\htdocs\intermodular\app\Http\Controllers\AuthController.php(24): App\Core\Auth::attempt(Array)
#7 C:\xampp\htdocs\intermodular\public\auth\login.php(30): App\Http\Controllers\AuthController->login(Object(App\Core\Request))
#8 {main}SQLSTATE[42S02]: Base table or view not found: 1146 Table 'jardineria.clients' doesn't exist
#0 C:\xampp\htdocs\intermodular\app\Core\DB.php(161): PDO->prepare('SELECT * FROM c...')
#1 C:\xampp\htdocs\intermodular\app\Core\DB.php(96): App\Core\DB::prepare('SELECT * FROM c...', Array)
#2 C:\xampp\htdocs\intermodular\app\Core\DB.php(129): App\Core\DB::select('App\\Models\\Clie...', 'SELECT * FROM c...', Array)
#3 C:\xampp\htdocs\intermodular\app\Core\QueryBuilder.php(114): App\Core\DB::selectOne('App\\Models\\Clie...', 'SELECT * FROM c...', Array)
#4 C:\xampp\htdocs\intermodular\app\Http\Validators\RegisterValidator.php(39): App\Core\QueryBuilder->first()
#5 C:\xampp\htdocs\intermodular\public\auth\register.php(27): App\Http\Validators\RegisterValidator::validate(Object(App\Core\Request))
#6 {main}SQLSTATE[42S02]: Base table or view not found: 1146 Table 'jardineria.clients' doesn't exist
#0 C:\xampp\htdocs\intermodular\app\Core\DB.php(161): PDO->prepare('SELECT * FROM c...')
#1 C:\xampp\htdocs\intermodular\app\Core\DB.php(96): App\Core\DB::prepare('SELECT * FROM c...', Array)
#2 C:\xampp\htdocs\intermodular\app\Core\DB.php(129): App\Core\DB::select('App\\Models\\Clie...', 'SELECT * FROM c...', Array)
#3 C:\xampp\htdocs\intermodular\app\Core\QueryBuilder.php(114): App\Core\DB::selectOne('App\\Models\\Clie...', 'SELECT * FROM c...', Array)
#4 C:\xampp\htdocs\intermodular\app\Http\Validators\RegisterValidator.php(39): App\Core\QueryBuilder->first()
#5 C:\xampp\htdocs\intermodular\public\auth\register.php(27): App\Http\Validators\RegisterValidator::validate(Object(App\Core\Request))
#6 {main}Call to undefined method App\Http\Controllers\AuthController::showRegisterForm()
#0 {main}Call to undefined method App\Http\Controllers\AuthController::showRegisterForm()
#0 {main}Call to undefined method App\Http\Controllers\AuthController::showRegisterForm()
#0 {main}Call to undefined method App\Http\Controllers\AuthController::showRegisterForm()
#0 {main}Call to undefined method App\Http\Controllers\AuthController::showLoginForm()
#0 {main}Call to undefined method App\Http\Controllers\AuthController::showLoginForm()
#0 {main}Call to undefined method App\Http\Controllers\AuthController::showLoginForm()
#0 {main}SQLSTATE[42S02]: Base table or view not found: 1146 Table 'jardineria.users' doesn't exist
#0 C:\xampp\htdocs\intermodular\app\Core\DB.php(161): PDO->prepare('SELECT * FROM u...')
#1 C:\xampp\htdocs\intermodular\app\Core\DB.php(96): App\Core\DB::prepare('SELECT * FROM u...', Array)
#2 C:\xampp\htdocs\intermodular\app\Core\DB.php(129): App\Core\DB::select('App\\Models\\User', 'SELECT * FROM u...', Array)
#3 C:\xampp\htdocs\intermodular\app\Core\QueryBuilder.php(114): App\Core\DB::selectOne('App\\Models\\User', 'SELECT * FROM u...', Array)
#4 C:\xampp\htdocs\intermodular\app\Models\User.php(62): App\Core\QueryBuilder->first()
#5 C:\xampp\htdocs\intermodular\app\Http\Validators\RegisterValidator.php(31): App\Models\User::findByUsername('Root')
#6 C:\xampp\htdocs\intermodular\public\auth\register.php(27): App\Http\Validators\RegisterValidator::validate(Object(App\Core\Request))
#7 {main}SQLSTATE[42S02]: Base table or view not found: 1146 Table 'jardineria.users' doesn't exist
#0 C:\xampp\htdocs\intermodular\app\Core\DB.php(161): PDO->prepare('SELECT * FROM u...')
#1 C:\xampp\htdocs\intermodular\app\Core\DB.php(96): App\Core\DB::prepare('SELECT * FROM u...', Array)
#2 C:\xampp\htdocs\intermodular\app\Core\DB.php(129): App\Core\DB::select('App\\Models\\User', 'SELECT * FROM u...', Array)
#3 C:\xampp\htdocs\intermodular\app\Core\QueryBuilder.php(114): App\Core\DB::selectOne('App\\Models\\User', 'SELECT * FROM u...', Array)
#4 C:\xampp\htdocs\intermodular\app\Models\User.php(62): App\Core\QueryBuilder->first()
#5 C:\xampp\htdocs\intermodular\app\Http\Validators\RegisterValidator.php(31): App\Models\User::findByUsername('Root')
#6 C:\xampp\htdocs\intermodular\public\auth\register.php(27): App\Http\Validators\RegisterValidator::validate(Object(App\Core\Request))
#7 {main}Class "App\Http\Middlewares\Security\CsrfMiddleware" not found
#0 {main}SQLSTATE[42S02]: Base table or view not found: 1146 Table 'jardineria.users' doesn't exist
#0 C:\xampp\htdocs\intermodular\app\Core\DB.php(161): PDO->prepare('SELECT * FROM u...')
#1 C:\xampp\htdocs\intermodular\app\Core\DB.php(96): App\Core\DB::prepare('SELECT * FROM u...', Array)
#2 C:\xampp\htdocs\intermodular\app\Core\DB.php(129): App\Core\DB::select('App\\Models\\User', 'SELECT * FROM u...', Array)
#3 C:\xampp\htdocs\intermodular\app\Core\QueryBuilder.php(114): App\Core\DB::selectOne('App\\Models\\User', 'SELECT * FROM u...', Array)
#4 C:\xampp\htdocs\intermodular\app\Models\User.php(62): App\Core\QueryBuilder->first()
#5 C:\xampp\htdocs\intermodular\app\Http\Validators\RegisterValidator.php(31): App\Models\User::findByUsername('Root')
#6 C:\xampp\htdocs\intermodular\public\auth\register.php(27): App\Http\Validators\RegisterValidator::validate(Object(App\Core\Request))
#7 {main}Call to undefined function url()
#0 C:\xampp\htdocs\intermodular\app\Core\Response.php(75): require()
#1 C:\xampp\htdocs\intermodular\app\Core\Response.php(29): App\Core\Response::loadView('auth.register', Array)
#2 C:\xampp\htdocs\intermodular\app\Core\helpers.php(101): App\Core\Response::view('auth.register', Array)
#3 C:\xampp\htdocs\intermodular\app\Http\Controllers\AuthController.php(47): view('auth.register')
#4 C:\xampp\htdocs\intermodular\public\auth\show-register.php(14): App\Http\Controllers\AuthController->showRegisterForm()
#5 {main}Call to undefined function url()
#0 C:\xampp\htdocs\intermodular\app\Core\Response.php(75): require()
#1 C:\xampp\htdocs\intermodular\app\Core\Response.php(29): App\Core\Response::loadView('auth.login', Array)
#2 C:\xampp\htdocs\intermodular\app\Core\helpers.php(101): App\Core\Response::view('auth.login', Array)
#3 C:\xampp\htdocs\intermodular\app\Http\Controllers\AuthController.php(17): view('auth.login')
#4 C:\xampp\htdocs\intermodular\public\auth\show-login.php(17): App\Http\Controllers\AuthController->showLoginForm()
#5 {main}Call to undefined function url()
#0 C:\xampp\htdocs\intermodular\app\Core\Response.php(75): require()
#1 C:\xampp\htdocs\intermodular\app\Core\Response.php(29): App\Core\Response::loadView('auth.login', Array)
#2 C:\xampp\htdocs\intermodular\app\Core\helpers.php(101): App\Core\Response::view('auth.login', Array)
#3 C:\xampp\htdocs\intermodular\app\Http\Controllers\AuthController.php(17): view('auth.login')
#4 C:\xampp\htdocs\intermodular\public\auth\show-login.php(17): App\Http\Controllers\AuthController->showLoginForm()
#5 {main}Call to undefined method App\Core\Request::input()
#0 C:\xampp\htdocs\intermodular\public\auth\register.php(27): App\Http\Validators\RegisterValidator::validate(Object(App\Core\Request))
#1 {main}Call to undefined method App\Core\Request::input()
#0 C:\xampp\htdocs\intermodular\public\auth\login.php(27): App\Http\Validators\LoginValidator::validate(Object(App\Core\Request))
#1 {main}Call to undefined method App\Core\Request::input()
#0 C:\xampp\htdocs\intermodular\public\auth\login.php(27): App\Http\Validators\LoginValidator::validate(Object(App\Core\Request))
#1 {main}